tests/AppBundle/EventListener/CommentNotificationListenerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Tests\AppBundle\EventListener;
+
+use AppBundle\Entity\Comment;
+use AppBundle\Entity\Post;
+use AppBundle\Entity\User;
+use AppBundle\EventListener\CommentNotificationListener;
+use Symfony\Component\EventDispatcher\GenericEvent;
+use Symfony\Component\Routing\Generator\UrlGeneratorInterface;
+use Symfony\Component\Translation\TranslatorInterface;
+
+/**
+ * Unit test for the event listener
+ *
+ * See http://symfony.com/doc/current/book/testing.html#unit-tests
+ *
+ * Execute the application tests using this command (requires PHPUnit to be installed):
+ *
+ *     $ cd your-symfony-project/
+ *     $ ./vendor/bin/phpunit
+ */
+class CommentNotificationListenerTest extends \PHPUnit_Framework_TestCase
+{
+    public function testOnCommentCreated()
+    {
+        $mailerProphecy = $this->prophesize(\Swift_Mailer::class);
```

Comment:
It is not possible to have the same Swift_Message instance, the test cannot pass.
Swift_Message should be injected.

```
$message = \Swift_Message::newInstance()
    ...
 ;

 $mailerProphecy->send($message)->shouldBeCalled();
```